extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/supervisor/KafkaSupervisor.java

```diff
@@ -1420,7 +1432,7 @@ public Void apply(@Nullable List<Void> input)
           {
             return null;
           }
-        }, workerExec
+        }
```

Comment:
It may also be fine in a lot of situations to just cast it, since presumably nobody is actually trying to cast the returned Void value to a Void. But, eh.